internal/commands/user/delete.go

```diff
@@ -60,10 +55,12 @@ func (cmd *CommandDelete) Handler(profile *cli.Profile, ui terminal.UI) error {
 	if appErr != nil {
 		return appErr
 	}
-	users, usersErr := cmd.inputs.resolveUsers(ui, cmd.realmClient, app)
-	if usersErr != nil {
-		return usersErr
+
+	users, userErr := cmd.inputs.resolveUsers(ui, cmd.realmClient, app)
```

Comment:
[meganit] i think this diff goes away if you just rename this back to `usersErr`...

Reply:
[meganit] idiomatic go prefers `errXXX`, or simply, `err` 😎

Reply:
I went with err for all 3 calls in the Handlers of the commands (and I even refactored user list a bit to line up), lmk of any nits on the naming here :)